### PR TITLE
docs: Adding documentation for concatAll operator.

### PR DESCRIPTION
**Pull request by sumitarora**
Fixes: https://github.com/ReactiveX/rxjs-docs/issues/64

### CODE REVIEW

**Comment by sumitarora** [comment]
@ashwin-sureshkumar @ladyleet If this is what's needed for docs for operator's I will start doing PR's for others too 😄

**Comment by btroncone** [comment]
@ladyleet Any objections?

**Comment by ashwin-sureshkumar** [comment]
@btroncone - In the existing docs there is a `return` type specified for operator which is missing in this doc, is that by design or we need to add that section ?

**Comment by btroncone** [comment]
I don't think it's necessary, to me it's covered by the operator signature. `eg public combineAll(project: function): Observable`.

File: src/operator-docs/combination/concatAll.ts [hunk 1]
```diff
@@ -1,6 +1,67 @@
...
+    "combineAll",
+    "concat",
+    "concatMap",
+    "concatMapTo",
+    "exhaust",
+    "mergeAll",
+    "switch",
+    "zipAll"
+  ],
+  additionalResources: []
```
Comment (ashwin-sureshkumar) [inline src/operator-docs/combination/concatAll.ts]: @sumitarora - please add links to `sourceCode` and `specs`

**Comment by sumitarora** [comment]
@btroncone @ashwin-sureshkumar One thing I noticed is the `walkthrough.extras` section is not getting rendered. Is someone working on that?

**Comment by btroncone** [comment]
@sumitarora  Creating a PR for this right now.

**Comment by ladyleet** [comment]
LGTM! Thx @sumitarora and thx @ashwin-sureshkumar for getting Sumit involved! :) Thx @btroncone for reviewing the PR.

**Pull request merged**